### DISCUSSION TITLE
Sharing `submitter` property on `up:form:submit` event object

### DISCUSSION THREAD

**Discussion by thetutlage** [unpoly/unpoly#318] [Ideas]
The native form submission event has a property called [submitter](https://developer.mozilla.org/en-US/docs/Web/API/SubmitEvent/submitter) which is a reference to the form submit button.

Should we share the same reference on `up:form:submit` event as well?

**Comment by triskweline** [comment]
This is part of the upcoming 2.5 release, although the property will be called `{ submitButton }` (eb62dc7).

What's already possible with 2.4 is, when your button has a `[name]` and `[value]` attribute, it will be included in `event.renderOptions.params`.

**Reply by thetutlage** [reply]
Thanks for all the work you guys are putting in Unpoly :)
